tools/wake-format/emitter.cpp

```diff
@@ -1666,7 +1666,7 @@ wcl::doc Emitter::walk_import(ctx_t ctx, CSTElement node) {
 
 wcl::doc Emitter::walk_interpolate(ctx_t ctx, CSTElement node) {
   MEMO(ctx, node);
-  MEMO_RET(walk_placeholder(ctx, node));
+  MEMO_RET(walk_placeholder(ctx.binop(), node));
```

Comment:
lol this is kind of a dirty trick, we should add a TODO: to generalize this to a "don't split" member instead because thats what it really is.

Reply:
Yeah, the  name is incorrect, and pretty much was from the beginning 😓